### DISCUSSION TITLE
Upgrade 4.0.8 to 5.0.2?

### DISCUSSION THREAD

**Discussion by 73blazer** [LycheeOrg/Lychee#2188] [Q&A] [answered]
Can I just run migrate from 5.0.2 code base or do I need to upgrade to some interim releases first like 4.7 or 4.13 ?

**Comment by ildyria** [comment, marked as answer]
In theory no. It should be pretty seamless.
I would recommend you to do a backup of your DB before doing so for the following reasons:
- if update fails you can revert.
- it gives us a failing case to work on (should you want to share your data) :)

**Comment by 73blazer** [comment]
Thanks for you response! Backup is a no brainer, it's all backed up,, www, the pics, database, config, mac hine images... off site too with daily snapshots! 
 I did  just read in a post a few down that subdirectory hosting is not in 5, mine is in website.com/pics so I'm guessing that qualifies as subdirectory hosting. I'll do 4.13 for now I think. Thanks!

**Comment by 73blazer** [comment]
I upgraded to 4.13 for now.  I do have to say, upgrade from 4.0.8 to 4.13 while initially there was an issue probably due mostly to confusion on my part, went perfectly seamlessly.  As an IT admin, as a DB2 admin, I know that takes some work for upgrades like that to happen. That's commendable. I thank you and the Lychee team for your work.
